libs/_example/tests/CMakeLists.txt

```diff
@@ -4,28 +4,31 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 if (NOT HPX_WITH_TESTS)
+  hpx_set_option(HPX__EXAMPLE_WITH_TESTS VALUE OFF FORCE)
```

Comment:
I'm considering removing the example module altogether now that we have a few actual modules in place. Documentation about creating modules should probably anyway go into the developer documentation. Do you think there's value in keeping it?

Reply:
Yah, let's remove it. Should be done in a separate PR, though.